src/liballoc/vec.rs

```diff
@@ -53,15 +53,15 @@
 //! [`Index`]: ../../std/ops/trait.Index.html
 //! [`IndexMut`]: ../../std/ops/trait.IndexMut.html
 //! [`vec!`]: ../../std/macro.vec.html
-
+// ignore-tidy-filelength
```

Comment:
?

Reply:
It means that the file went over 3000 LOC which tidy doesn't like. Ideally the file should be refactored into smaller pieces.

Reply:
Any recommendations what should be split out?

Reply:
As a start, the iterators (e.g. Drain) could probably be moved out to a module:
```
src/liballoc/vec/
    - mod.rs
    - iter.rs
```
but it's probably best to do the split in a different PR to keep the diff limited.